src/paginate.spec.ts

```diff
@@ -1361,26 +1406,6 @@ describe('paginate', () => {
         expect(result.links.current).toBe('?page=1&limit=20&sortBy=id:ASC&filter.age=$not:$null')
     })
 
-    it('should return result based on not null query', async () => {
```

Comment:
why has this case been removed?

Reply:
duplicate of the upper one... or am i wrong ?

Reply:
seems right, good catch then :)